src/containers/Accounts/AMM/AMMAccounts/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { useParams, useRouteMatch } from 'react-router'
+import { useTranslation } from 'react-i18next'
+import '../../styles.scss'
+import formatBalance from '../../../../rippled/lib/txSummary/formatAmount'
+import {
+  getAccountTransactions,
+  getAMMInfo,
+} from '../../../../rippled/lib/rippled'
+import { Tabs } from '../../../shared/components/Tabs'
+import { analytics, ANALYTIC_TYPES, formatAsset } from '../../../shared/utils'
+import SocketContext from '../../../shared/SocketContext'
+import { ERROR_MESSAGES } from '../../Errors'
+import NoMatch from '../../../NoMatch'
+import {
+  AMMAccountHeader,
+  AmmDataType,
+} from './AMMAccountHeader/AMMAccountHeader'
+import { AccountTransactionTable } from '../../AccountTransactionTable'
+
+const getErrorMessage = (error: string) =>
+  ERROR_MESSAGES[error] || ERROR_MESSAGES.default
+
+export const AMMAccounts = (props: any) => {
```

Comment:
Why is this not typed?

Reply:
was only used for language which has been removed. removing this.